<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsultaAlugueis</class>
 <widget class="QDialog" name="ConsultaAlugueis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>-70</y>
     <width>20</width>
     <height>591</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>20</width>
     <height>541</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>171</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Alugueis em Aberto:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Listar todos 
Alugueis em Aberto</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>341</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Nome do Cliente:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>1</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ForbiddenCursor</cursorShape>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <column>
       <property name="text">
        <string>Nome</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Quantidade</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Valor</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Preço do Total do Aluguel:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Preco">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>BlankCursor</cursorShape>
      </property>
      <property name="text">
       <string>R$</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="DevolucaoButton">
      <property name="text">
       <string>CONFIRMAR DEVOLUÇÃO</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>VOLTAR</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
